### ISSUE TITLE
Basic PHP to Java sending demo, select invoice from Nextcloud and recipient from Addressbook

### ISSUE THREAD

**Issue by michielbdejong** [pondersource/peppol-php#141]

**Comment by yasharpm** [comment]
How do you select invoice from Nextcloud? Does it mean forwarding one of the bills that we have already received? It would probably be easier to allow the user to create a new invoice since we already have some UI done for it.

**Comment by michielbdejong** [comment]
Yes! That's the idea. The user would compose an invoice in the UI, adding a recipient and some line items.
